### DISCUSSION TITLE
Problems downloading prerequisites for EWITool

### DISCUSSION THREAD

**Discussion by nomorereeds** [community/community#37157] [New to GitHub]
1) Downloaded Java 8 successfully but cannot see it anywhere.
2) When trying to download CoreMidi4j-1.6 (1).jar error message pops up
"Cannot be opened because it is from an unidentified developer"
MacBook Pro Catalina version 10.15.7

**Comment by ossanna16** [comment]
Hi @nomorereeds,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Java. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by nomorereeds** [reply]
Actually I found Java 8 today, it has apparently installed correctly. The problem is as I reported "When trying to download CoreMidi4j-1.6 (1).jar error message pops up"Cannot be opened because it is from an unidentified developer"
MacBook Pro Catalina version 10.15.7

I assume I should be searching for "Java communities" or such? I'm downloading these programs so I can run EWITools with my Akai EWI 4000 as its editing program, Vyzex, is no longer supported. An EWITools community would likely be helpful but I can't find one. Any suggestion would be helpful.

**Reply by ossanna16** [reply]
Hi there @nomorereeds! Yes, I suggest looking for forums that specifically handle Java as a topic. We definitely have Java users here but most users will stop by to ask or answer GitHub related questions so you will have more luck getting fast responses in a Java related forum.
